uber/templates/preregistration/preregbase.html

```diff
@@ -11,10 +11,10 @@
     var checkCap = function () {
         $.getJSON('check_prereg', function(check) {
             if (check.force_refresh) {
-                location.reload(); // Reload the page to prevent registering after reg is closed
+                location.reload();  // Reload the page to prevent registering after reg is closed
             }
         });
     };
```

Comment:
15 seconds seems unnecessarily agressive, and this was cluttering our logs, so I upped it to 90 seconds.